firmware/arduino/chessboard/uart.ino

```diff
@@ -150,7 +151,8 @@ bool validateMessageFromPi(volatile char * message)
     if ((message[ITYPE_IDX] != ALIGN_AXIS         ||  message[EXTRA_IDX] < '0'  ||  message[EXTRA_IDX] > '4')  &&  
         (message[ITYPE_IDX] != SET_ELECTROMAGNET  ||  message[EXTRA_IDX] < '0'  ||  message[EXTRA_IDX] > '1')  &&
         (message[ITYPE_IDX] != RETRANSMIT) &&
-        (message[ITYPE_IDX] != SET_HUMAN_MOVE_VALID || message[EXTRA_IDX] < '0' || message[EXTRA_IDX] > '1')
+        (message[ITYPE_IDX] != SET_HUMAN_MOVE_VALID || message[EXTRA_IDX] < '0' || message[EXTRA_IDX] > '1')   &&
+        (message[ITYPE_IDX] != ENABLE_MOTORS  ||  message[EXTRA_IDX] < '0'  ||  message[EXTRA_IDX] > '1')      &&
```

Comment:
You have an extra `&&` here, but instead of removing it can u add in the check for software reset as above and same thing for below in `makeMove`